Configuration/odin-views.json

```diff
@@ -216,6 +217,18 @@
                     }
                 }]
             }
+        },
+        "views/odin/app": {
```

Comment:
I think we should start to split these apart. Maybe put this in a new `odin-app.json` file, or some such similar name?